guides/common/modules/proc_installing-the-satellite-server-packages.adoc

```diff
@@ -32,7 +32,6 @@ include::proc_installing-server-packages-el.adoc[]
 endif::[]
 
 ifdef::foreman-el,katello,satellite[]
-== [[rhel-7]]{RHEL} 7
```

Comment:
Wouldn't you instead remove satellite from the line above and leave this line as is?

Reply:
Ohh yes. I was occupied more with git mess. I will be considering it.

Reply:
You did NOT change this.

Reply:
I am waiting for some time in case there are comments from other reviewers. If it's done, I am ready to make changes and push.

Reply:
Resolved

Reply:
This wasn't resolved. The Satellite docs now render with an empty "Installing packages" section:
![image](https://user-images.githubusercontent.com/94284/172339110-4fbd159e-3d73-4c7c-9344-5d717b1041f0.png)

Moreso, this document didn't gain a RHEL8 section at all?!

Reply:
@mjivraja